### DISCUSSION TITLE
How do I get to unlink my email address from my github account?

### DISCUSSION THREAD

**Discussion by megarocha** [community/community#80963] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I keep trying to unlink my email other github account, but it keeps saying GitHub support will review your request within 1-3 days. Isn't unlinking an email address not needed for Github support to review?

**Comment by ananuness** [comment]
Hii! You can go to `Settings` > `Emails` and see that you have a **primary** email, if the one you want to remove is the primary, you can add a new email, verify it and then delete the other one!
